tests/unit/meta/Dimensions.ts

```diff
@@ -0,0 +1,119 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { WidgetBase } from '../../../src/WidgetBase';
+import { v } from '../../../src/d';
+import { ProjectorMixin } from '../../../src/main';
+import Dimensions from '../../../src/meta/Dimensions';
+import { stub } from 'sinon';
+import global from '@dojo/core/global';
+
+let rAF: any;
+
+function resolveRAF() {
+	for (let i = 0; i < rAF.callCount; i++) {
+		rAF.getCall(0).args[0]();
+	}
+	rAF.reset();
+}
+
+registerSuite({
+	name: 'meta - Dimensions',
+
+	beforeEach() {
+		rAF = stub(global, 'requestAnimationFrame');
+	},
+
+	afterEach() {
+		rAF.restore();
+	},
+
+	'dimensions are blank on first render, set on second'(this: any) {
```

Comment:
Perhaps something like `dimensions are correct configured`? (even if in the test we do two renders, etc?)